qadence/serialization.py

```diff
@@ -94,11 +198,11 @@ def serialize(obj: SUPPORTED_TYPES, save_params: bool = False) -> dict:
     """
     Supported Types:
 
-    AbstractBlock | QuantumCircuit | QuantumModel | TransformedModule | Register | Module
+    AbstractBlock | QuantumCircuit | QuantumModel | torch.nn.Module | Register | Module
     Serializes a qadence object to a dictionary.
 
     Arguments:
-        obj (AbstractBlock | QuantumCircuit | QuantumModel | Register | Module):
+        obj (AbstractBlock | QuantumCircuit | QuantumModel | Register | torch.nn.Module):
```

Comment:
Why replacing `TransformedModule` with torch ones ?

Reply:
Because we're getting rid of `QNN` and `TransformedModule` on `qadence`, so they cannot be explicitly referenced. `QuantumModel` and `torch.nn.Module` are the base classes for every new model added to the serialization (it was the plan from moving the models out of qadence).

Reply:
OK makes sense. In that case, I suggest to import them:

```
from torch import nn.Modules
```

To make the type annotations more readable.

Reply:
I wanted to make explicit that `Module` is a `torch` object on the docs and type annotations by using `torch.nn.Module`, so we don't get confused by `qadence`'s own objects, but I can change that of course.